testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -36,7 +36,8 @@ def setUp(self):
         self.universe_rev = MDAnalysis.Universe(PSF, DCD)
         self.universes = [self.universe1, self.universe2, self.universe_rev]
         self.psa = MDAnalysis.analysis.psa.PSAnalysis(self.universes,           \
-                                               path_select='name CA')
+                                               path_select='name CA',           \
+                                               targetdir=tempdir.TempDir())
```

Comment:
`TempDir()` is a context manager and I think if you use it like `TemopDir().name` (which would give you the directory name) then the directory will be created and immediately destroyed.
Either wrap the whole thing in
```python
with tempdir.in_tempdir():
```
or use `tempfile.mkdtemp()` explicitly together with explicit clean up in `tearDown()`.